internal/lib/lib.go

```diff
@@ -1337,11 +1337,19 @@ func PassthroughShardSize() uint32 {
 	}
 	return 1
 }
-
+func GetAKOID_Prefix() string {
```

Comment:
Nit: Can we not use the underscore in function name, also lets reuse this same function in GetShardVSPrefix too.

Reply:
addressed